salt/log/setup.py

```diff
@@ -486,7 +488,8 @@ def setup_console_logger(log_level='error', log_format=None, date_format=None):
             # There's already a logging handler outputting to sys.stderr
```

Comment:
Right above this line we check to see if any handler has the stream set to `sys.stderr` we should instead, with this new behavior, compare it to `get_console_handler_stream()`
```diff
-         if handler.stream is sys.stderr:
+         if handler.stream is get_console_handler_stream():
```